### DISCUSSION TITLE
Mobile app?

### DISCUSSION THREAD

**Discussion by Else00** [goniszewski/grimoire#136] [Q&A]
I often find myself needing to access and manage my bookmarks on the go. While the current Grimoire platform works great on desktop, it would be beneficial to have a mobile solution that integrates seamlessly with Grimoire's features, especially for quick bookmarking and categorization while away from the computer.

I'm wondering if there are any existing mobile bookmark manager apps (for Android and iOS) that are already compatible with Grimoire's Integration API. If not, are there any plans to either support existing mobile apps or develop a dedicated mobile app for Grimoire? Having such an option would greatly enhance the flexibility and convenience of using Grimoire on mobile devices.

It would be helpful to know if there is a list of recommended mobile bookmark managers compatible with Grimoire's API, or if there's any ongoing development for a native mobile app.

**Comment by goniszewski** [comment]
Hello @Else00! 

At this time, only the official browser extension uses this API. I have plans to improve mobile UI experience for Grimoire. Near this time, also PWA support will be introduced, so you will be able to "install" it on your device.

Regarding a dedicated mobile application, my experience is limited to creating some simple React Native app with Expo, and that knowledge is _significantly outdated_. So, I would need to explore better ways of implementing a mobile app rather than developing it entirely from the ground up.

Are you familiar with any good mobile apps that allow for custom API URLs for their backend?

**Comment by rodrigohpalmeirim** [comment]
@goniszewski check out [Capacitor](https://capacitorjs.com/). I've used it before and I think it would fit this project really well. It's extremely easy to integrate into an existing web app. If you wanted to, you could even share the codebase between the web version and the mobile version.
